R/links_secuTrial.R

```diff
@@ -4,7 +4,7 @@
 #' Understanding the links can be difficult. This function produces a map linking the forms
 #' with common variables.
 #'
-#' @param object secuTrialdata object
+#' @param object \code{secuTrialdata} object
```

Comment:
first object not needed

Reply:
First object is the parameter name so I think its correct as is even though it looks a bit weird.

Reply:
ah, ok